src/App.test.js

```diff
@@ -3,6 +3,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/KPINETWORKS IS UNDER CONSTRUCTION/i);
+  const linkElement = screen.getByText(/KPINETWORK IS UNDER CONSTRUCTION/i);
```

Comment:
since that this text will change soon, I think that maybe we should just validate that a string exists